app/src/main/res/values/bools.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <bool name="pref_default_value_enable_local_time">false</bool>
     <bool name="pref_default_value_enable_notification">true</bool>
+    <bool name="pref_default_value_enable_hide_bottom_navigation">false</bool>
```

Comment:
I think we can change default value after when it does not work fine. So I think we can do it `true` 😄

Reply:
Ok, I'll change it to `true` 😄